frontend/app/routes/public/apply/$id/child/review-adult-information.tsx

```diff
@@ -63,12 +62,10 @@ export async function loader({ context: { configProvider, serviceProvider, sessi
   const { ENABLED_FEATURES, HCAPTCHA_SITE_KEY } = getEnv();
   const t = await getFixedT(request, handle.i18nNamespaces);
   const locale = getLocale(request);
-  const lookupService = getLookupService();
 
   // Getting province by Id
-  const allRegions = localizeRegions(lookupService.getAllRegions(), locale);
-  const provinceMailing = allRegions.find((region) => region.provinceTerritoryStateId === state.contactInformation.mailingProvince);
-  const provinceHome = allRegions.find((region) => region.provinceTerritoryStateId === state.contactInformation.homeProvince);
+  const provinceMailingProvinceTerritoryStateAbbr = state.contactInformation.mailingProvince ? serviceProvider.getProvinceTerritoryStateService().findById(state.contactInformation.mailingProvince)?.abbr : undefined;
+  const provinceHomeProvinceTerritoryStateAbbr = state.contactInformation.homeProvince ? serviceProvider.getProvinceTerritoryStateService().findById(state.contactInformation.homeProvince)?.abbr : undefined;
```

Comment:
Might be a nitpick but I feel "province" doesn't need to be in the variable name twice.

```suggestion
  const mailingProvinceTerritoryStateAbbr = state.contactInformation.mailingProvince ? serviceProvider.getProvinceTerritoryStateService().findById(state.contactInformation.mailingProvince)?.abbr : undefined;
  const homeProvinceTerritoryStateAbbr = state.contactInformation.homeProvince ? serviceProvider.getProvinceTerritoryStateService().findById(state.contactInformation.homeProvince)?.abbr : undefined;
```

This local variable naming applies to other routes in this PR

Reply:
Fixed